src/constructs/ec2/vpc.ts

```diff
@@ -7,7 +7,9 @@ interface VpcFromIdProps extends Omit<VpcAttributes, "availabilityZones"> {
   availabilityZones?: string[];
 }
 
-type VpcFromIdParameterProps = Omit<VpcFromIdProps, "vpcId">;
+interface VpcFromIdParameterProps extends Omit<VpcFromIdProps, "vpcId"> {
```

Comment:
Could `VpcFromIdParameterProps` extend [`AppIdentity`](https://github.com/guardian/cdk/blob/d69bdbe181d90f0f2c502dee9b05095f1402301a/src/constructs/core/identity.ts#L9-L11) instead?

Reply:
As discussed IRL, this is not a trivial change and so we've elected to leave this for a future PR.